.github/workflows/ci.yml

```diff
@@ -35,12 +35,15 @@ jobs:
       matrix:
         include:
           - name: Linux
+            id: linux
             os: ubuntu-22.04-16core
             flags: ""
           - name: Linux Nightly
+            id: linux_nightly
```

Comment:
This I don’t think is actually something we want to do – the action only caches the dependencies of the workspace, which are largely the same between the two configurations. This artifact is also pretty large, which coupled with the 10GiB repository-wide cache limit can mean that with too many different configurations we won’t be retaining the caches for long enough for them to get reused before they are overwritten.

Reply:
Oh right my brain bugged, I thought "nightly" here meant rustc nightly, not nightly-features… this is now fixed :)